src/utils/feature-flags.ts

```diff
@@ -1,67 +1,80 @@
-import { initialize, LDFlagSet } from 'launchdarkly-js-client-sdk'
+import { initialize, LDClient, LDFlagSet, LDOptions, LDUser } from 'launchdarkly-js-client-sdk'
 
-const defaultFlagSet = {
-  'correction-ui-enabled': true,
-  'alteration-ui-enabled': true
+// get rid of "element implicitly has an 'any' type..."
+declare const window: any
+
+/**
+ * Default flag values when LD is not available.
+ * Uses "business-edit" project (per LD client id in config).
+ */
+const defaultFlagSet: LDFlagSet = {
+  'correction-ui-enabled': false,
+  'alteration-ui-enabled': false
```

Comment:
I have changed this so that the feature is DISABLED if the user cannot connect to LD. This will prevent users from accessing features accidentally.
(The only impact is that you may need to override these when running locally.)